### ISSUE TITLE
requirements.txt missing

### ISSUE THREAD

**Issue by dpriskorn** [Ammarpad/OutreachyProject#2]

**Comment by Ammarpad** [comment]
Yes I have this on mind. I am still working on the scripts though and things may change greatly but thanks for the reminder. I will close this issue when the repo is more stable and the file is added.
